android/libraries/rib-base/src/main/java/com/badoo/ribs/core/Interactor.kt

```diff
@@ -39,8 +39,8 @@ import java.util.UUID
  * @param <C> the type of Configuration this Interactor can expect to push to its [Router].
  * @param <V> the type of [RibView].
  **/
-abstract class Interactor<C : Parcelable, V : RibView>(
-    protected val router: Router<C, V>,
+abstract class Interactor<C : Parcelable, Content : C, Overlay : C, V : RibView>(
```

Comment:
Can we somehow reduce amount of parameters that we need to pass for Interactor and Router?
It would be super annoying to pass 5 parameters for router and 4 for Interactor.
Maybe we can merge Content, Overlay, PermanentPart etc.

Reply:
I'd like it, but I don't see a way unfortunately. We can discuss in person.